tests/nftPrice.test.ts

```diff
@@ -0,0 +1,161 @@
+      "./tests/mocks/cnft-api-input.json"
+    ) as any;
+
+    Object.keys(currentCNFTsPrice).forEach((key) => {
+      nockEndpoint(`./tests/mocks/cnft/${key}.json`, getCNFTURL(key));
+    });
+
+    try {
+      currentCNFTsPrice = await getCollectionsUsingOpenCNFTInterval(
+        currentCNFTsPrice,
+        0,
+        2
+      );
+      expect(
+        currentCNFTsPrice[
+          "b97859c71e4e73af3ae83c30a3172c434c43041f6ff19c297fb76094"
+        ]
+      ).toStrictEqual({
+        id: 63760,
+        name: "EarthNode",
+        policies: "b97859c71e4e73af3ae83c30a3172c434c43041f6ff19c297fb76094",
+        data: {
+          attribution:
+            "NOTICE: © 2022 OpenCNFT. Just include the source of the data (opencnft.io) ;)",
+          policy: "b97859c71e4e73af3ae83c30a3172c434c43041f6ff19c297fb76094",
+          thumbnail: "ipfs://QmYUA3THPRhQcdmSzgJUTYMa98hvYWtD9i9vUG8ZcNeQeo",
+          total_volume: 2745353000000,
+          first_sale: 1.7976931348623157e308,
+          total_tx: 33,
+          total_assets_sold: 32,
+          asset_minted: 1000,
+          asset_holders: 515,
+          highest_sale: {
+            asset_name: "EarthNode835",
+            name: "b97859c71e4e73af3ae83c30a3172c434c43041f6ff19c297fb76094.EarthNode835",
+            price: 135000000000,
+          },
+          floor_price: 78500000000,
+          floor_price_marketplace: "jpg.store",
+        },
+        lastUpdatedTimestamp:
+          currentCNFTsPrice[
+            "b97859c71e4e73af3ae83c30a3172c434c43041f6ff19c297fb76094"
+          ].lastUpdatedTimestamp,
+      });
+
+      expect(
+        currentCNFTsPrice[
+          "b863bc7369f46136ac1048adb2fa7dae3af944c3bbb2be2f216a8d4f"
+        ]
+      ).toStrictEqual({
+        id: 27835,
+        name: "Berry",
+        policies: "b863bc7369f46136ac1048adb2fa7dae3af944c3bbb2be2f216a8d4f",
+        data: {
+          attribution:
+            "NOTICE: © 2022 OpenCNFT. Just include the source of the data (opencnft.io) ;)",
+          policy: "b863bc7369f46136ac1048adb2fa7dae3af944c3bbb2be2f216a8d4f",
+          thumbnail:
+            "ipfs://ipfs/Qmb43sxTNDVze7VAodazKoPDaB6vjMKs8sEqGuexiAu5TS",
+          total_volume: 1186259990000,
+          first_sale: 1.7976931348623157e308,
+          total_tx: 16,
+          total_assets_sold: 16,
+          asset_minted: 100,
+          asset_holders: 84,
+          highest_sale: {
+            asset_name: "BerryCardinal",
+            name: "b863bc7369f46136ac1048adb2fa7dae3af944c3bbb2be2f216a8d4f.BerryCardinal",
+            price: 115000000000,
+          },
+          floor_price: 64000000000,
+          floor_price_marketplace: "jpg.store",
+        },
+        lastUpdatedTimestamp:
+          currentCNFTsPrice[
+            "b863bc7369f46136ac1048adb2fa7dae3af944c3bbb2be2f216a8d4f"
+          ].lastUpdatedTimestamp,
+      });
+    } catch (e) {
+      throw e;
+    }
+  });
+
+  // TODO: fix the nock error for this test
```

Comment:
TODO?

Reply:
yeah for another PR. I couldn't figure it out and it's not required. No idea why Nock is acting weird